service-worker/index.js

```diff
@@ -29,6 +31,9 @@ self.addEventListener('activate', (event) => {
 });
 
 self.addEventListener('fetch', (event) => {
+  const strategy = STRATEGY;
```

Comment:
It seems unnecessary to declare constants and assign them to the imported values, specially since other imported values are used directly in this very same code.